Sketch.java

```diff
@@ -1,36 +1,206 @@
 import processing.core.PApplet;
 
+/**
+ * Description: This program define arrays to animate a collection of objects.
+ * @author: Stephanie Tam
+ */
+
 public class Sketch extends PApplet {
-	
-	
+
+  // Global variables
+  int intCircleYSpeed = 1;
+  int intPlayerX = 200;
+  int intPlayerY = 300;
+  int intPlayerLives = 3;
+
+  boolean blnPressDown = false;
+  boolean blnPressUp = false;
+  boolean blnCharW = false;
+  boolean blnCharA = false;
+  boolean blnCharS = false;
+  boolean blnCharD = false;
+
   /**
    * Called once at the beginning of execution, put your size all in this method
    */
+  
   public void settings() {
 	// put your size call here
     size(400, 400);
   }
 
+  // Blank array used throughout the program
+  float[] circleY = new float[25];
+  float[] circleX= new float[25];
+  boolean[] ballHideStatus = new boolean[25];
+
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
+
   public void setup() {
-    background(210, 255, 173);
+
+    // Array to store snowball random X values
+    for(int index = 0; index < circleX.length; index++){
+      circleX[index] = random(width);
+    }
+
+    // Array to store snowball random Y values
+    for(int index = 0; index < circleY.length; index++){
+      circleY[index] = random(height - 150);
+    }
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
+
+  /**
+   * Draws snowball
+   * 
+   * @param blnPressDown boolean of down arrow key pressed
+   * @param blnPressUp boolean of up arrow key pressed
+   * @param intCircleYSpeed y speed of circle
+   * @param circleX x-coordinate of the ellipse
+   * @param circleY y-coordinate of the ellipse
+   * @param intPlayerX x-coordinate of user
+   * @param intPlayerY y-coordinate of user
+   * @param blnCharW boolean of W key pressed
+   * @param blnCharA boolean of A key pressed
+   * @param blnCharS boolean of S key pressed
+   * @param blnCharD boolean of D key pressed
```

Comment:
no `@params` needed because the `draw` method doesn't have parameters in its definition